ipywidgets/widgets/widget_media.py

```diff
@@ -159,8 +159,19 @@ class Image(_Media):
 
     # Define the custom state properties to sync with the front-end
     format = Unicode('png', help="The format of the image.").tag(sync=True)
-    width = CUnicode(help="Width of the image in pixels.").tag(sync=True)
-    height = CUnicode(help="Height of the image in pixels.").tag(sync=True)
+    width = CUnicode(help="Width of the image in pixels. Use layout.width "
+                          "for styling the widget.").tag(sync=True)
+    height = CUnicode(help="Height of the image in pixels. Use layout.height "
+                           "for styling the widget.").tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super(Image, self).__init__(*args, **kwargs)
+        # This is done to make explicit a CSS rule that was applied
+        # to all img tags, including the Image widget, in the
+        # classic notebook. Removing it in the future would not
+        # be unreasonable.
+        self.layout.max_width = '100%'
+        self.layout.height = 'auto'
```

Comment:
There will be a conflict here if I supply a layout in args/kwargs. Maybe this should only be set if the layout used is the default one?